src/Microsoft.PowerShell.Commands.Utility/resources/NewObjectStrings.resx

```diff
@@ -144,6 +144,12 @@
   <data name="CannotCreateTypeConstrainedLanguage" xml:space="preserve">
     <value>Cannot create type. Only core types are supported in this language mode.</value>
   </data>
+  <data name="CannotCreateTypeNoLanguage" xml:space="preserve">
+    <value>Cannot create type. Only core types are supported in NoLanguage mode on a policy locked down machine.</value>
+  </data>
+  <data name="CannotCreateTypeRestrictedLanguage" xml:space="preserve">
+    <value>Cannot create type. Only core types are supported in restricted language mode on a policy locked down machine.</value>
+  </data>
```

Comment:
I would expect these to either be of the same form, or to use a templatised format (depending on whether we localise the "language mode" names).

So something like:

```xml
  <data name="CannotCreateTypeLanguageMode" xml:space="preserve">
    <value>Cannot create type. Only core types are supported in {0} language mode on a policy locked down machine.</value>
  </data>
```

And then supply the name of the language mode like:

```csharp
string.Format(NewObjectStrings.CannotCreateTypeLanguageMode, "NoLanguage")
...
string.Format(NewObjectStrings.CannotCreateTypeLanguageMode, "RestrictedLanguage")
```

Reply:
From the docs it seems we don't do localise language mode names:

![image](https://user-images.githubusercontent.com/7009879/101065616-37813a00-354a-11eb-88fb-e7418d037260.png)

Ideally though, rather than raw strings, we should use something like the enum name. I've done this in the suggestion above.